packages/server/graphql/mutations/helpers/notifySlack.ts

```diff
@@ -97,11 +104,54 @@ export const startSlackMeeting = async (
     utm_campaign: 'invitations'
   }
   const options = {searchParams}
-  const team = await dataLoader.get('teams').load(teamId)
-
+  const [team, meeting] = await Promise.all([
+    dataLoader.get('teams').load(teamId),
+    dataLoader.get('newMeetings').load(meetingId)
+  ])
   const meetingUrl = makeAppURL(appOrigin, `meet/${meetingId}`, options)
-  const slackText = `${team.name} has started a meeting!\n To join, click here: ${meetingUrl}`
-  notifySlack('meetingStart', dataLoader, teamId, slackText).catch(console.log)
+  const button = {text: 'Join meeting', url: meetingUrl, type: 'primary'} as const
+  const blocks = [
+    makeSection('Meeting started :wave: '),
+    makeSections([`*Team:*\n${team.name}`, `*Meeting:*\n${meeting.name}`]),
+    makeSection(`*Link:*\n<${meetingUrl}|https:/prbl.in/${meetingId}>`),
```

Comment:
is `https:/prbl.in/${meetingId}` valid?

Reply:
The url isn't valid but it's just a hyperlink. When the user clicks on it, they'll be redirected to the regular longer link with UTM params. The url could be anything but I kept it in the same format as our team invite link. 

If we want to use a valid short url using our existing format of `https:/prbl.in/${token}`, we'd run into a security issue as the token works as an invite to a team and a user could send the Slack notification to a public channel

Reply:
feels a little weird that the link label is an invalid url. could we instead write something like: [Take me there](https://www.google.com)?

Reply:
For sure! I thought this was the approach your were suggesting [here](https://parabol.slack.com/archives/D017D764FDY/p1621887268010200).

The "Join meeting" button already takes us to the link so perhaps we just drop the "Link" section?

<img width="455" alt="notification" src="https://user-images.githubusercontent.com/39854876/120848526-82e8c500-c53a-11eb-9f0b-4b52bd70708e.png">

Reply:
>The "Join meeting" button already takes us to the link so perhaps we just drop the "Link" section?

@nickoferrall I added the link section on purpose because I think having the link there could be handy for cases when you need to share the link with someone, for example on a text message, or an email. You can't copy the link from the "Joing meeting" button, unless you join the meeting first and then copy the link from within Parabol, which I thought would be a bit annoying if joining the meeting is not what you want to do at the moment.